Extension/src/LanguageServer/extension.ts

```diff
@@ -246,6 +246,77 @@ export async function activate(): Promise<void> {
         clients.timeTelemetryCollector.setFirstFile(activeEditor.document.uri);
         activeDocument = activeEditor.document;
     }
+
+    await setupCppChatVariable(util.extensionContext);
+}
+
+export async function setupCppChatVariable(context: vscode.ExtensionContext | undefined): Promise<void>
+{
+    if (!context) {
+        return;
+    }
+
+    vscode.chat.registerChatVariableResolver('cpp',
```

Comment:
Will we have a way to measure usage of `#cpp`? I'm wondering if we should add our own telemetry, or if VS Code will emit some automatically.

Reply:
usage telemetry added